ac_s.lua

```diff
@@ -941,6 +941,26 @@ Citizen.CreateThread(function()
 		end
 	end)
 	
+	AddEventHandler("giveWeaponEvent", function(source, data)
+		source = tonumber(source)
+		local entity = NetworkGetEntityFromNetworkId(data.pedId)
+		if DoesEntityExist(entity) then
+			local owner = NetworkGetEntityOwner(entity)
+			if owner ~= source then
+				CancelEvent()
```

Comment:
should the event always be cancelled? despite the component being off?

Reply:
Fixed